vmdb/app/helpers/container_helper/textual_summary.rb

```diff
@@ -13,6 +13,22 @@ def textual_group_properties
   #
 
   def textual_image
-    {:label => "image", :value => @record.image}
+    {:label => "Image", :value => @record.image}
+  end
+
+  def textual_name
+    {:label => "Name", :value => @record.name}
+  end
+
+  def textual_state
+    {:label => "State", :value => @record.state}
+  end
+
+  def textual_restart_count
+    {:label => "Restart Count", :value => @record.restart_count}
+  end
+
+  def textual_container_id
+    {:label => "Container ID", :value => @record.container_id}
```

Comment:
Per our discussion, this container_id is not a relationship and we do want to keep as it as part of the summary